examples/typescript-backbone/js/app.ts

```diff
@@ -319,6 +369,12 @@ class AppView extends Backbone.View {
 				completed: completed,
 				remaining: remaining
 			}));
+
+			this.$('.filters li a')
+				.removeClass('selected')
+				.filter('[href="#/' + (taskFilter || '') + '"]')
```

Comment:
Yeah, there were several things I would do differently including a switch to DefinitelyTyped, but it started to become more of a rewrite and I wasn't up for that. I guess the original author would argue this is a manual form of "DOM diffing" optimization but if so it's a premature one. Even the raw jQuery example doesn't do it, plus it seems like it's not scalable.